lib/diagnostic.rb

```diff
@@ -5,36 +5,36 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.max_by(&:length)
```

Comment:
May work. But you need to use the comparison operator since it's a comparison. 

`list.max { |a, b| a.length <=> b.length }`